### ISSUE TITLE
Add Skip to Main Content button

### ISSUE THREAD

**Issue by echappen** [cloud-gov/cg-ui#328]
For a11y, add a Skip to Main Content button. USWDS should have one to reference in its [header component](https://designsystem.digital.gov/components/header/).

### Acceptance Criteria
- [x] Skip button is first item to be focused when tabbing through the site
- [x] Skip button is hidden from view unless focused
- [x] Clicking the button brings focus to `<main>` element
